hypha/apply/projects/templates/application_projects/vendor_success.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base-apply.html" %}
+{% load static %}
+
+{% block title %}Contractor Setup Completed{% endblock %}
```

Comment:
Please make all strings translatable via i18n, see e.g. https://github.com/HyphaApp/hypha/blob/60c90d888f24af87f6644d62f145704cc08fe745/hypha/apply/funds/templates/funds/application_base.html#L36

Reply:
This goes for all string in all templates, or rather all string any where.